src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="#" method="post">
+      <div class="group-form-field">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="form-field">
+              Surname:
+            <input type="text"
+              autocomplete="off"
+              name="Surname"
+              required
+            >
+          </label>
```

Comment:
```suggestion
          <label class="form-field">
            Surname:
            <input type="text"
              autocomplete="off"
              name="Surname"
              required
            >
          </label>
```
this element is on the same level as input
fix everywhere